task03/task03/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task03
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int x;
+            x = int.Parse(Console.ReadLine());
+
+            int a = x % 10;
+            Console.WriteLine("число единиц равно " + a);
+
+            int b = x / 1000;
+            Console.WriteLine("число тысяч равно " + b);
+
+            int c = (x % 100) / 10;
+            
+            int d = (x / 100) % 10; 
```

Comment:
Операции умножения, деления и вычисления остатка имеют одинаковый приоритет, поэтому, будучи записанные без скобок, будут выполняться в порядке слева направо, то есть в ваших выражениях скобки можно не писать. Хотя употребление скобок тоже приемлемо.
```suggestion
            int c = x % 100 / 10;
            
            int d = x / 100 % 10; 
```